nfs/src/rpc_task.cpp

```diff
@@ -2008,6 +2019,26 @@ static void read_callback(
     // Free the context.
     delete ctx;
 
+    if (status != 0) {
+        if (NFS_STATUS(res) == NFS3ERR_JUKEBOX) {
+            task->get_client()->jukebox_retry(task);
+            return;
+        } else {
```

Comment:
nit, can we club this with if (status == 0) above so that the code pattern looks similar to other APIs.
i.e.

 if (status == 0) 
 {
} else if (NFS_STATUS(res) == NFS3ERR_JUKEBOX) {
} else {
}